From 9d15cf34e02ea7d5596efec2ded96186467d51a8 Mon Sep 17 00:00:00 2001
From: Kirill Kapranov <kirill.kapranov@compulab.co.il>
Date: Tue, 10 May 2022 19:22:49 +0300
Subject: [PATCH 41/41] dts:phy: Eliminate excessive reset

The eth phy is tuned on u-boot, however the reset at Linux boot cleans all the
tuning and makes the phy inoperable. Eliminate the reset at Linux boot so as to
preserve the phy tuned.

Signed-off-by: Kirill Kapranov <kirill.kapranov@compulab.co.il>
---
 arch/arm64/boot/dts/compulab/ucm-imx8m-mini.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/compulab/ucm-imx8m-mini.dtsi b/arch/arm64/boot/dts/compulab/ucm-imx8m-mini.dtsi
index 377574898946..c40bc84e3e17 100644
--- a/arch/arm64/boot/dts/compulab/ucm-imx8m-mini.dtsi
+++ b/arch/arm64/boot/dts/compulab/ucm-imx8m-mini.dtsi
@@ -79,6 +79,7 @@
 		regulator-always-on;
 		enable-active-high;
 		startup-delay-us = <500>;
+		regulator-boot-on;
 	};
 
 	usdhc1_pwrseq: usdhc1_pwrseq {
-- 
2.11.0

